lib/ood_core/job/adapters/lsf/helper.rb

```diff
@@ -92,6 +92,14 @@ def batch_submit_args(script, after: [], afterok: [], afternotok: [], afterany:
     args += ["-W", (script.wall_time / 60).to_i] unless script.wall_time.nil?
     args += ["-L", script.shell_path.to_s] unless script.shell_path.nil?
 
+    # Skip if nil
+    env = script.job_environment || {}
+    if script.copy_environment?
+      args += ["-env", (["all"] + env.keys).join(",")]
+    elsif script.copy_environment? == false
```

Comment:
I know tests catch this, but could you add a short comment here that we are explicitly comparing to false because in the nil case we don't want to add any args?